engine/src/main/resources/org/hibernate/validator/ValidationMessages_es.properties

```diff
@@ -1,53 +1,53 @@
+org.hibernate.validator.constraints.ModCheck.message                = El d\u00edgito de comprobaci\u00f3n para ${validatedValue} no es v\u00e1lido, ha fallado la suma de comprobaci\u00f3n ${modType}
+org.hibernate.validator.constraints.NotBlank.message                = no puede estar vac\u00edo
+org.hibernate.validator.constraints.NotEmpty.message                = no puede estar vac\u00edo
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = la expresi\u00f3n de script "{script}" no se ha evaluado en verdadera
+org.hibernate.validator.constraints.Range.message                   = debe estar entre {min} y {max}
+org.hibernate.validator.constraints.SafeHtml.message                = puede tener contenido HTML no seguro
+org.hibernate.validator.constraints.ScriptAssert.message            = la expresi\u00f3n de script "{script}" no se ha evaluado en verdadera
+org.hibernate.validator.constraints.UniqueElements.message          = solo debe contener elementos exclusivos
+org.hibernate.validator.constraints.URL.message                     = debe ser un URL v\u00e1lido
 
-org.hibernate.validator.constraints.br.CNPJ.message                 = n\u00FAmero de identificaci\u00F3n tributario corporativo Brazile\u00F1o (CNPJ) inv\u00E1lido
-org.hibernate.validator.constraints.br.CPF.message                  = n\u00FAmero de identificaci\u00F3n tributario individual Brazile\u00F1o (CPF) inv\u00E1lido
-org.hibernate.validator.constraints.br.TituloEleitoral.message      = n\u00FAmero de identificaci\u00F3n electoral Brazile\u00F1o inv\u00E1lido
+org.hibernate.validator.constraints.br.CNPJ.message                 = n\u00famero de registro de contribuyente corporativo de Brasil no v\u00e1lido (CNPJ)
+org.hibernate.validator.constraints.br.CPF.message                  = n\u00famero de registro de contribuyente individual de Brasil no v\u00e1lido (CPF)
+org.hibernate.validator.constraints.br.TituloEleitoral.message      = n\u00famero de tarjeta de ID de votante de Brasil no v\u00e1lido
 
-org.hibernate.validator.constraints.pl.REGON.message                = n\u00FAmero de identificaci\u00F3n tributario Polaco (REGON) inv\u00E1lido
-org.hibernate.validator.constraints.pl.NIP.message                  = n\u00FAmero de identificaci\u00F3n VAT (NIP) inv\u00E1lido
-org.hibernate.validator.constraints.pl.PESEL.message                = n\u00FAmero de identificaci\u00F3n nacional Polaco (PESEL) inv\u00E1lido
+org.hibernate.validator.constraints.pl.REGON.message                = N\u00famero de identificaci\u00f3n de contribuyente de Polonia no v\u00e1lido (REGON)
+org.hibernate.validator.constraints.pl.NIP.message                  = N\u00famero de identificaci\u00f3n de VAT no v\u00e1lido (NIP)
+org.hibernate.validator.constraints.pl.PESEL.message                = N\u00famero de identificaci\u00f3n nacional de Polonia no v\u00e1lido (PESEL)
```

Comment:
Same remark for the consistency of the acronym. Either stick to capital for each word of the acronym translation or use lowercase.